### ISSUE TITLE
[P1][Bug][a11y] Focus order of "customization" bottom sheet is incorrect

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ods-ios#703]
### Prerequisites

- [X] I have [searched the backlog](https://github.com/Orange-OpenSource/ods-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14 Plus

### OS Version

17.1.1

### App version

v0.18.0

### Describe the issue

Inside almost all variants, when there's a "customization" bottom sheet, in most of cases, the focus order is not correct
Sometimes the first element focused is "customization" (instead of page title or main description), sometimes elements inside the sheet are ordered inconsistently.
![image](https://github.com/Orange-OpenSource/ods-ios/assets/105045667/5b514304-d012-4cb1-b4c2-1d96b4b1f1ad)

NB: behavior is correct on variant Buttons>Icons and Sliders demo

### Expected Behavior

On each variant page, focus order should be :

1. back button
2. colour palette
3 all interactive elements in demo view (focus forced on first element of this view on page loading)
5. customization sheet, and following elements in order
6. tab bar

**Comment by pya35** [comment]
OK sur 0.18 (1708889908)
